src/Docker/Runner/UsageFile/NullUsageFile.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Keboola\DockerBundle\Docker\Runner\UsageFile;
+
+class NullUsageFile implements UsageFileInterface
+{
+    public function __construct()
+    {
+    }
+
+    public function setDataDir($dataDir)
+    {
+    }
+
+    public function setFormat($format)
+    {
+    }
+
+    public function setJobId($jobId)
```

Comment:
`jobId` by nemusel bejt součástí interface? používá se pouze s jobmapperem, ne?

Reply:
no to sis odpovedel hned v nasledujicim commentu, pouziva se to uvnitr Runner, kterej nevi, co v UsageFile ma (jestli NullUsageFile nebo UsageFile) a timpadem to musi byt soucasti interfacu :/